ros/angel_msgs/msg/ObjectDetection2dSet.msg

```diff
@@ -37,3 +37,9 @@ float32[] bottom
 # This is a flattened 2D row-major matrix with shape:
 #   [num_detections, len(label_vec)]
 float64[] label_confidences
+
+# Vector length of each descriptor.
+int32 descriptor_dims
```

Comment:
Nitpick: should this be singular? This is describing just the consistent length dimension of a single descriptor vector.